### PR TITLE
chore: update readme example

### PR DESCRIPTION
**Pull request by wegi8**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,8 +44,9 @@ Is the same as:
 <script setup>
 import _imports_0 from '../image.png'
 </script>
-
-<img src="_imports_0" />
+<template>
+  <img :src="_imports_0" />
+ </template>
```
Comment (patak-dev) [inline README.md:49]: There is an extra space
```suggestion
<template>
  <img :src="_imports_0" />
</template>
```

**Review by patak-dev — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -44,8 +44,9 @@ Is the same as:
 <script setup>
 import _imports_0 from '../image.png'
 </script>
-
-<img src="_imports_0" />
+<template>
+  <img :src="_imports_0" />
+</template>
```
[the remark this answers is not in the document]
Comment (wegi8) [reply README.md:49]: Okay, I'll take a fresh look at it all.

**Comment by wegi8** [comment]
@patak-dev Thank you. I'm sorry I didn't correct it carefully.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (wegi8) [reply README.md]: it's OK

**Review by patak-dev — APPROVED** [review]

**Pull request merged**